nav2_planner/src/planner_server.cpp

```diff
@@ -522,6 +529,56 @@ PlannerServer::publishPlan(const nav_msgs::msg::Path & path)
   }
 }
 
+void PlannerServer::isPathValid(
+  const std::shared_ptr<nav2_msgs::srv::IsPathValid::Request> request,
+  std::shared_ptr<nav2_msgs::srv::IsPathValid::Response> response)
+{
+  unsigned int mx = 0;
```

Comment:
Move these closer to where they are used, L568-ish or so

Reply:
Done